helm/prometheus-rules/templates/alerting-rules/app.rules.yml

```diff
@@ -124,6 +138,20 @@ spec:
         sig: none
         team: batman
         topic: releng
+    - alert: WorkloadClusterAppPendingUpdate
+      annotations:
+        description: 'Current version of {{`App {{ $labels.name }} is {{ $labels.deployed_version }} but it should be {{ $labels.version }}.`}}'
+      expr: label_replace(app_operator_app_info{catalog!~"control-plane-.*", deployed_version!="", status="deployed", version_mismatch="true"}, "cluster_id", "$1", "namespace", "(([^g]|g[^i]|gi[^a]|gia[^n]|gian[^t]|giant[^s]|giants[^w]|giantsw[^a]|giantswa[^r]|giantswar[^m])*)") == 1
```

Comment:
This is where we need the replace so the workload cluster ID is shown in the alerts.